source.py

```diff
@@ -26,22 +26,29 @@ def fetchAll():
     print(process.stdout)
 
 def getCommitHash(parentTicketNumber):
-    #returns parent ticket commit hash by rev-parse of parent ticket branch
-    result = subprocess.run(['git', 'rev-parse', 'DOCSP-'+parentTicketNumber], capture_output=True, text=True)
-    print("REV PARSE")
+    #returns parent ticket commit hash by rev-list of HEAD
```

Comment:
made this change because `rev-parse` of parent ticket branch was returning most recent commit for parent ticket on local branch, not squashed commit from master.